bin/bundle

```diff
@@ -54,14 +54,14 @@ m = Module.new do
   def lockfile_version
     return unless File.file?(lockfile)
     lockfile_contents = File.read(lockfile)
-    return unless lockfile_contents =~ /\n\nBUNDLED WITH\n\s{2,}(#{Gem::Version::VERSION_PATTERN})\n/
+    return unless lockfile_contents =~ /\n\nBUNDLED WITH\n\s{2,}(#{Gem::Version::VERSION_PATTERN})\n/o
```

Comment:
Same question here.

Reply:
@joemasilotti - same with this, a suggestion from standardrb

```bin/bundle:57:40: Performance/ConstantRegexp: Extract this regexp into a constant, memoize it, or append an `/o` option to its options.```